modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -719,6 +738,65 @@ private void _addObjectDefinitionResources(
 		}
 	}
 
+	private Set<String> _getAccountEntryRestrictiveObjectRelationshipsNames(
```

Comment:
@devpedrotavares please rename this methdo to `_getAccountEntryRestrictedObjectRelationshipNames` (`Restricted` instead of `Restrictive`) to match the  `AccountEntryRestricted` parameter of the `ObjectDefinition` model.